Code/Tools/SceneAPI/SceneCore/Events/ExportProductList.cpp

```diff
@@ -49,6 +51,39 @@ namespace AZ
                 return *this;
             }
 
+            void ExportProductList::Reflect(ReflectContext* context)
+            {
+                if (auto* behaviorContext = azrtti_cast<AZ::BehaviorContext*>(context))
+                {
+                    behaviorContext->Class<ExportProduct>("ExportProduct")
```

Comment:
This should have a version reflected, so it can be updated when the behavior context serialization changes.

Reply:
this is the behavior context that I do not think has a version and the behavior context does not serialize anything, right?

Reply:
You need to put the version in the serialize context in that case. It's a little awkward but it's still helpful as a signal to people who use these systems.

Reply:
This class is never serialized since it is only used to collect the products that will be exported plus dependency information. Adding a serialization context just to have a version number would be misleading.

Reply:
We use the serialization context just for version numbers all over the place, like in the AssImp logic https://github.com/o3de/o3de/blob/3ea3d7a436255856ec8a4be0be62739c83aec72a/Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpMeshImporter.cpp#L35

Reply:
I would argue that those are not needed unless the asset pipeline uses this to force rebuilds of product assets that would be affected by the version number such as if bumping that AssImp number would force all FBX/STL files to rebuild.

Reply:
That's how those are used for AssImp. For scripting we've used just a version on the serialize context to signify a behavior context change, so that script users know they may need to update their scripts based on that change.

Reply:
Okay, I will get that done